### PR TITLE
Stub transport

### PR DESCRIPTION
**Pull request by deejay1**
Bases on work done in PR #145 and adds a retryable zipkin API client in tests.

### CODE REVIEW

**Comment by deejay1** [comment]
CC @gugu

**Comment by adriancole** [comment]
you all are really adding interesting features. very under-appreciated
library

File: aiozipkin/tracer.py [hunk 1]
```diff
@@ -96,7 +96,10 @@ def create(zipkin_address: str,
 
     async def build_tracer() -> Tracer:
         sampler = Sampler(sample_rate=sample_rate)
-        transport = Transport(zipkin_address, send_interval=send_interval, loop=loop)
+        if not zipkin_address:
+            transport = cast(Transport, StubTransport())
```
Comment (deejay1) [inline aiozipkin/tracer.py]: this shouldn't be needed anymore, I'll just make Trace accept TransportABC, but I'll try to add some tests to it first...

File: tests/test_zipkin.py [hunk 2]
```diff
@@ -2,12 +2,26 @@
...
 import pytest
 import aiozipkin as az
 from yarl import URL
 
 
+async def _retry_zipkin_client(url: URL, client: 'aiohttp.ClientSession',
+                               retries: int = 5) -> Dict:
+    tries = 0
+    while tries < retries:
+        asyncio.sleep(5)
```
Comment (jettify) [inline tests/test_zipkin.py]: Probably good idea to add sleep time to the function arguments, as we have for `retries`, also lets make it lower say 1, just to speed up tests a bit.

File: aiozipkin/tracer.py (hunk 1 above)
Comment (jettify) [reply aiozipkin/tracer.py]: Agreed cast should be removed. 

Lets think about API should we create new function something like:
```python
def create_with_transprot(
    transport: OptTransport: None,,
    local_endpoint: Endpoint,
    loop: OptLoop=None) -> Awaitable[Tracer]:
```
?

File: aiozipkin/transport.py [hunk 3]
```diff
@@ -9,7 +11,32 @@
...
+        pass
+
+    @abc.abstractmethod
+    async def close(self) -> None:
+        """Performs additional cleanup actions if required."""
+        pass
+
+
+class StubTransport(TransportABC):
+    def __init__(self, *args: Optional[Any], **kwargs: Optional[Any]) -> None:
```
Comment (jettify) [inline aiozipkin/transport.py]: may be add `maxlen` to there arguments with default value 100?

File: aiozipkin/tracer.py (hunk 1 above)
Comment (deejay1) [reply aiozipkin/tracer.py]: Looks good, but maybe as a separate PR ?

Comment (jettify) [reply aiozipkin/tracer.py]: agreed

**Pull request merged**